core/src/main/java/io/kestra/plugin/core/execution/Labels.java

```diff
@@ -43,23 +43,23 @@
             title = "Add labels based on a webhook payload",
             full = true,
             code = {
-                "id: webhook-based-labels",
+                "id: webhook_based_labels",
                 "namespace: company.team",
                 "tasks:",
-                "  - id: update-labels-with-map",
+                "  - id: update_labels_with_map",
                 "    type: io.kestra.plugin.core.execution.Labels",
                 "    labels:",
-                "      customerId: \"{{trigger.body.customerId}}\"",
-                "  - id: by-list",
+                "      customerId: \"{{ trigger.body.customerId }}\"",
+                "  - id: by_list",
                 "    type: io.kestra.plugin.core.execution.Labels",
                 "    labels:",
-                "      - key: orderId",
-                "        value: \"{{trigger.body.orderId}}\"",
-                "      - key: orderType",
-                "        value: \"{{trigger.body.orderType}}\"",
```

Comment:
@shrutimantri double check the expressions here now you've renamed some of the tasks